### PR TITLE
simplified attributes

### PR DESCRIPTION
**Pull request by kristinaspring**
fixes https://github.com/xmidt-org/bascule/issues/63

### CODE REVIEW

**Review by johnabass — APPROVED** [review]
Nice and simple, good idea there

File: attributes.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+var nilTime = time.Time{}
+
+type BasicAttributes map[string]interface{}
+
+func (a BasicAttributes) Get(key string) (interface{}, bool) {
+	v, ok := a[key]
+	return v, ok
+}
+
```
Comment (johnabass) [inline attributes.go:17]: Is `BasicAttributes` supposed to implement `Attributes`? If so, just this one method isn't enough.  Looks like `Attributes` has several methods?

Comment (kristinaspring) [reply attributes.go:17]: Nope, `Attributes` just has one method.

**Review by johnabass — APPROVED** [review]
Looks great!

**Pull request merged**